docker-compose.yml

```
@@ -134,7 +134,7 @@ services:
         - "${TMPDIR:-/tmp/localstack}:/tmp/localstack"
         - "/var/run/docker.sock:/var/run/docker.sock"
     consul:
-        image: consul:latest
+        image: consul:1.15.4
```

```suggestion
        image: hashicorp/consul:latest
```

We should be using the updated supported image, not the deprecated image.

---

You are right! Thanks!

---

Ah, I fixed that in #1853 for consul CI. OK.